src/aio/clientsideCallbacks.js

```diff
@@ -0,0 +1,63 @@
+window.dash_clientside = window.dash_clientside || {};
+
+window.dash_clientside.themeSwitch = {
+
+    applyDbcClass: function (_) {
+        let dash_container = document.querySelector('#_dash-app-content')
+
+        // create a set with existing classes (if there are)
+        let classNameSet = new Set(dash_container.className.split(" "))
+        // add the dbc class if not already in the set
+        classNameSet.add('dbc')
+        // set the container className
+        dash_container.className = [...classNameSet].join(' ').trim()
+        return window.dash_clientside.no_update
+    },
```

Comment:
This is a nice idea, but it would be a breaking change and should to be removed.  Not everyone uses the dbc class when using this component.